src/GitInfo/readme.txt

```diff
@@ -92,3 +92,9 @@ Available MSBuild customizations:
                             whether the branch and tags (if any) 
                             will be used to find a base version.
                             Defaults to empty value (no ignoring).
+
+$(GitTagRegex): Regular Experssion used with git describe to find the BaseTag
+           Defaults to * (all)
+           
+$(GitBaseVersionExpr): Regular Experssion used with git describe to find the BaseTag
```

Comment:
Typo in both strings (`Experssion` -> `Expression`). Also, it's not correct that this regex is used to find BaseTag. It's used to parse the base version too.